src/HttpExponentialBackoff.js

```diff
@@ -127,6 +127,25 @@ class HttpExponentialBackoff {
       return timeToWait
     }
   }
+
+  /**
+   * Retry Delay returns a function that either:
+   *  - return the value of Retry-After header, if present
+   *  - implements exponential backoff otherwise
+   *
+   * @param {number} initialDelayInMillis The multiplying factor and the initial delay. Eg. 100 would mean the retries would be spaced at 100, 200, 400, .. ms
+   * @returns {Function} retryDelayFunction {function(*=, *, *): number}
+   * @private
+   */
+  __getRetryDelayWithRetryAfterHeader (initialDelayInMillis) {
+    return (attempt, error, response) => {
+      const retryAfter = response.headers.get('Retry-After')
+      if (retryAfter != null) {
+        return parseRetryAfterHeader(retryAfter)
```

Comment:
we can add a logger.debug here similar to https://github.com/adobe/aio-lib-core-networking/pull/22/commits/94b1d84e38306b6d263d4f06277a832c478efd6f#diff-d612dfd0110171e13c29ac011a0ca33d478c896fabba06a00822d8bc131f9753R125